.github/workflows/close-inactive-issues.yml

```diff
@@ -0,0 +1,24 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/stale@v3
+        with:
+          any-of-labels: "Status: Waiting for Customer"
```

Comment:
FYI: I assume this label must exist.

For managing labels, we use this config file: https://github.com/commercetools/ui-kit/blob/main/.github-labelsrc.json
I guess we can go with the black color

https://github.com/commercetools/ui-kit/blob/25e3e2f7e3d4b231705e89592ef6bbf306864701/.github-labelsrc.json#L160-L161

There is a command `yarn labels:sync` that creates the github labels based on the config. You need to have a `AUTH_TOKEN` environment variable to make the command work. See https://github.com/commercetools/github-labels/

Reply:
Yeap, thanks, the label should be present for sure. Don't you think, creating it [directly in the repo](https://github.com/commercetools/ui-kit/issues/labels) is enough?

Reply:
Added new label